pulp.spec

```diff
@@ -426,6 +426,13 @@ then
   pulp-gen-ca-certificate
 fi
 
+if [ $1 -eq 2 ]; # an upgrade
+then
+  %if %{pulp_systemd} == 1
+    /sbin/systemctl daemon-reload > /dev/null 2>&1
+  %endif
+fi
+
```

Comment:
can you confirm this works ok on el6?

```bash
if [ 1 -eq 1 ];
then
  echo "HELLO"
fi
```

the above works, but if I comment out the echo statement, it fails. You may need to wrap the if statement in the %if macro.